src/com/google/JsComp.gwt.xml

```diff
@@ -0,0 +1,12 @@
+<module rename-to="jscomp">
+  <inherits name="com.google.gwt.user.User" />
+  <inherits name="com.google.common.base.Base"/>
+  <inherits name="com.google.common.collect.Collect"/>
+  <source path="debugging/sourcemap" />
+  <source path="javascript/jscomp" excludes=".svn,.git,**/ant/**,**/refactoring/**,**/webservice/**,**/testing/**" />
+  <source path="javascript/rhino" excludes=".svn,.git,**/testing/**" />
+  <super-source path="javascript/jscomp/gwt/super" />
```

Comment:
I don't know how this is done internally at Google (I suppose it exists) but it looks to me like it could be clearer to put that file in `src/com/google/javascript/jscomp/gwt/` that follows the naming conventions of `client`, `super` and `public`; and add other `*.gwt.xml` files with `<source path="" />` inside `src/com/google/debugging/sourcemap` and `src/com/google/javascript/rhino` (and add the appropriate `<inherits/>` inside `src/com/google/javascript/jscomp/gwt/JsComp.gwt.xml`)